### ISSUE TITLE
Lights Out

### ISSUE THREAD

**Issue by kanak22** [Nivedita967/Data-Structure-and-Algorithms#680]
### Problem Link 
https://codeforces.com/problemset/problem/275/A

### Coding platform
Codeforces

### Programming language

- [ ] C
- [x] C++
- [ ] Java
- [x] Python

### Additional context (OPTIONAL)
I want to solve this in C++.

**Comment by ArkajyotiChakraborty** [comment]
Good Morning, Can i work on this issue.

**Comment by Nivedita967** [comment]
Hey @ArkajyotiChakraborty In which language you want to work?

**Comment by ArkajyotiChakraborty** [comment]
I want to work in Java.

**Comment by Nivedita967** [comment]
Sure @ArkajyotiChakraborty Go for it.

**Comment by JASKAMAL22** [comment]
I would like to work in python. Please assign it to me :)

**Comment by Nivedita967** [comment]
> I would like to work in python. Please assign it to me :)

Sure @JASKAMAL22 .

**Comment by Nivedita967** [comment]
@ArkajyotiChakraborty Are you working??

**Comment by ArkajyotiChakraborty** [comment]
yes will submit in couple of days, sorry having exams so balancing everything.

**Comment by Nivedita967** [comment]
> yes will submit in couple of days, sorry having exams so balancing everything.

No problem :)

**Comment by ArkajyotiChakraborty** [comment]
![image](https://user-images.githubusercontent.com/72427901/112633758-c4d0fd00-8e5f-11eb-964c-dd878efc42ae.png)

Hey, the solution got accepted what i need to do make get it merge,  new to open source.

**Comment by kanak22** [comment]
Hey @ArkajyotiChakraborty  you need to make a PR for getting it merged. You can google about how to make a pull request and can follow steps and learn by it and welcome to opensource :)

**Comment by ArkajyotiChakraborty** [comment]
ok will do it in completion of the other question.

**Comment by abhinav5481** [comment]
Hi @Nivedita967
Is this issue still open? Can I solve it in Java or c?
